Doc/extending/extending.rst

```diff
@@ -221,8 +221,14 @@ with an exception object (leaving out the error checking for now)::
            return NULL;
 
        SpamError = PyErr_NewException("spam.error", NULL, NULL);
-       Py_INCREF(SpamError);
-       PyModule_AddObject(m, "error", SpamError);
+       Py_XINCREF(SpamError);
+       if (PyModule_AddObject(m, "error", SpamError)) {
```

Comment:
I suggest to add also `< 0` in all other checks for `PyModule_AddObject()` result.

Reply:
Is this an agreed-upon style? I had always thought it was a matter of taste.

In either case, I guess all of these files should at least be consistent...

Reply:
Add the extra test with `< 0`